cea/inputlocator.py

```diff
@@ -49,6 +49,77 @@ def get_optimization_slave_results_folder(self):
         """
         return self._ensure_folder(self.get_optimization_results_folder(), "slave")
 
+    def get_optimization_slave_StorageOperationData(self, configkey):
+        """scenario/outputs/data/calibration/clustering/checkpoints/..."""
+        return os.path.join(self.get_optimization_slave_results_folder(),
+                            configkey + '_StorageOperationData.csv')
```

Comment:
concatenating strings always makes me nervous (say, `configkey` wasn't a string - what should happen?) why don't we do this instead: `'%(configkey)s_StorageOperationData.csv' % locals()`